docker-compose.yml

```diff
@@ -27,6 +27,11 @@ services:
       - postgresql
       - legacy-elasticsearch
       - elasticsearch
+    develop:
+      watch:
+        - action: sync
+          path: .
+          target: /app
```

Comment:
This includes the sync action which will copy files from the host to the container.

There's also a rebuild action which I think we want to use to run `manage.py collectstatic` and `esbuild` when js and css files change.